lib/galaxy/schema/__init__.py

```diff
@@ -1,13 +1,44 @@
-import typing
+from typing import (
+    Dict,
+    List,
+    Optional,
+)
 
 from pydantic import BaseModel
 
 
 class BootstrapAdminUser(BaseModel):
     id = 0
-    email: typing.Optional[str] = None
-    preferences: typing.Dict[str, str] = {}
+    email: Optional[str] = None
+    preferences: Dict[str, str] = {}
     bootstrap_admin_user = True
 
     def all_roles(*args) -> list:
         return []
+
+
+class Tour(BaseModel):
```

Comment:
Since those are probably only used with tours I'd probably put this with `lib/galaxy/tours` ?

Reply:
That was my first version - until the last PR.
Moving to `tours/schema.py` (as per discussion below)